sources/api/bootstrap-containers/src/main.rs

```diff
@@ -92,18 +92,21 @@ const ENV_FILE_DIR: &str = "/etc/bootstrap-containers";
 const DROPIN_FILE_DIR: &str = "/etc/systemd/system";
 const PERSISTENT_STORAGE_DIR: &str = "/local/bootstrap-containers";
 const DROP_IN_FILENAME: &str = "overrides.conf";
+const DEFAULT_CONFIG_PATH: &str = "/etc/bootstrap-containers.toml";
```

Comment:
```suggestion
const DEFAULT_CONFIG_PATH: &str = "/etc/bootstrap-containers/bootstrap-containers.toml";
```

And then add a mount unit like `packages/host-ctr/etc-host-containers.mount.in` to ensure that `/etc/bootstrap-containers` is labeled as `secret_t`.